Milad.css

```diff
@@ -0,0 +1,127 @@
+/* Reset some default styles */
+body, h1, h2, h3, p, ul, li {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: Comic sans MS, sans-serif;
+    background-color: #f0f0f0;
+    background-image: url(https://www.w3schools.com/css/lights600x400.jpg);
+  background-repeat: no-repeat;
+  background-size:  cover;
+  color: #FFFFFF;
+  height: 700px;
+  width: auto;
+  
+      
+      
+}
+
+
+header {
+    background-color: #FFC0CB;
+    color: #48D1CC;
+    text-align: center;
+    padding: 2rem 0;
+}
+
+nav {
+    background-color: rgba(0, 0, 0, 0.7);
+    color: #FFFFFF;
+    padding: 10px 0;
+}
+
+nav ul {
+    list-style: none;
+    display: flex;
+    justify-content: center;
```

Comment:
This is good! However, you might want to experiment with different options for justify-content, such as space-between, space-around, and space-evenly, to see if you prefer any of these over center.

But feel free to stick with the current option if you like it this way :)